fobi_custom/plugins/form_elements/fields/types.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+
+# Valid SurveyComponent types for the Data Analysis Designer (DAD). These are
+# the UIDs of Fobi form elements that can sensibly be turned into charts.
+
+# "Count" types represent a single numeric value, and will be displayed as a median.
+COUNT_TYPES = [
```

Comment:
The use case I'm thinking of here is specifically one where the user wants to compare the median of a survey to the medians of past surveys (not currently possible, but planned in #102). In that case, the chart would show a series of bars that are comparable.

Showing the count of each response seems dangerous to me since it's basically a free-response question and we could wind up with hundreds of slightly different responses. A variation of that approach might be to bin the data in some way and compare distributions, but then we'd have to know up front what the bins are, or bin them by e.g. quantiles and risking having quantiles that don't make sense if the data are very close together.

This seems like a case where a [box plot](https://www.highcharts.com/demo/box-plot) could be a good compromise: in the single-survey case it would at least give a sense of the distribution of the answers, and in the multiple-survey case it would allow for more detailed comparisons. What do you think?

Reply:
All makes sense! I can't quite picture the UI for #102 yet, but I'm comfortable leaving this question open until you get to that issue. For single surveys a box or scatter plot makes sense to me.